src/components/RegistrationFormBody.tsx

```diff
@@ -101,49 +102,89 @@ const RegistrationFormBody: React.FC<{ field: FormListFieldData }> = ({
       >
         <DatePicker />
       </Form.Item>
-
+      {isChildForm && (
+        <>
+          <Form.Item
+            // {...fieldWithoutKey}
+            name={[field.name, 'birthday']}
+            fieldKey={[field.fieldKey, 'birthday']}
+            label="Date of Birth"
+            rules={[
+              {
+                required: true,
+                message: 'Please input your date of birth.',
+              },
+            ]}
+          >
+            <DatePicker />
+          </Form.Item>
+          <Form.Item
+            // {...fieldWithoutKey}
+            name={[field.name, 'school']}
+            fieldKey={[field.fieldKey, 'school']}
+            label="School"
+            className="inline-block-half"
+            rules={[
+              { required: true, message: "Please input your child's school" },
+            ]}
+          >
+            <Input placeholder="School" />
+          </Form.Item>
+          <Form.Item
+            // {...fieldWithoutKey}
+            name={[field.name, 'grade']}
+            fieldKey={[field.fieldKey, 'grade']}
+            label="Grade"
+            className="inline-block-half"
+            rules={[
+              {
+                required: true,
+                message: "Please input your child's current grade",
+              },
+            ]}
+          >
+            <Input placeholder="Grade" />
+          </Form.Item>
+        </>
+      )}
       <Form.Item
-        {...fieldWithoutKey}
+        // {...fieldWithoutKey}
         name={[field.name, 'allergies']}
         fieldKey={[field.fieldKey, 'allergies']}
         label="Allergies (if applicable)"
       >
         <TextArea rows={3} placeholder="Allergies" />
       </Form.Item>
-
       <Form.Item
         name={[field.name, 'diagnosis']}
         fieldKey={[field.fieldKey, 'diagnosis']}
         label="Diagnosis"
       >
         <TextArea rows={1} placeholder="Diagnosis" />
       </Form.Item>
-
       <Form.Item
-        {...fieldWithoutKey}
+        // {...fieldWithoutKey}
         name={[field.name, 'medication']}
         fieldKey={[field.fieldKey, 'medication']}
         label="Medication (if applicable)"
       >
         <TextArea rows={1} placeholder="Medication" />
       </Form.Item>
-
       <Form.Item
-        {...fieldWithoutKey}
+        // {...fieldWithoutKey}
         name={[field.name, 'otherNotes']}
         fieldKey={[field.fieldKey, 'otherNotes']}
         label="Other Notes"
       >
         <TextArea rows={3} placeholder="Other Notes" />
       </Form.Item>
-
       <Form.Item
-        {...fieldWithoutKey}
-        name={[field.name, 'picture']}
-        fieldKey={[field.fieldKey, 'picture']}
+        // {...fieldWithoutKey}
+        name={[field.name, 'profilePicture']}
+        fieldKey={[field.fieldKey, 'profilePicture']}
         label="Upload Profile Picture"
       >
-        <Dragger>
+        <Dragger multiple={false} beforeUpload={() => false}>
```

Comment:
same comment as other dragger, use `noop` from lodash for beforeUpload if it's actually just a no-op